packages/net/http/_client_connection.pony

```diff
@@ -181,6 +181,14 @@ actor _ClientConnection is HTTPSession
     """
     None
 
+  be dispose() =>
+    """
+    Close the connection from the client end.
+    """
+    match _conn
+      | let c: TCPConnection => c.dispose()
```

Comment:
The pipe operator in a match pattern should be on the same indentation level as the `match` keyword, such as:
```pony
match foo
| bar => baz
end
```